docs/multi-datasource/client_management_design.md

```diff
@@ -174,7 +174,7 @@ This is for plugin to access data source client via request handler. For example
 
 ### 4.4 Refactor data plugin search module to call core API to get datasource client
 
-`Search strategy` is the low level API of data plugin search module. It retrieve clients and query OpenSearch. It needs to be refactored to switch between default client and datasource client, depending on whether a request is send to datasource or not.
+`Search strategy` is the low level API of data plugin search module. It retrieves clients and query OpenSearch. It needs to be refactored to switch between default client and datasource client, depending on whether a request is send to datasource or not.
```

Comment:
```suggestion
`Search strategy` is the low level API of data plugin search module. It retrieves clients and queries OpenSearch. It needs to be refactored to switch between the default client and the datasource client, depending on whether or not a request is sent to the datasource.
```

Reply:
I think we also split "data source" in above paragrahp